### DISCUSSION TITLE
Problem with bdfconv

### DISCUSSION THREAD

**Discussion by DanielRodrigues1** [olikraus/u8g2#1746] [Q&A] [answered]
I cant convert this bdf file, i keep getting the erro "Cant find bdf file: '7Segments.bdf' " do i have to put the file on a especific folder?

**Comment by olikraus** [comment]
It needs to be in the same folder as the executable. It is also possible to specify the full path of the bdf file.

**Reply by DanielRodrigues1** [reply]
weird i keep getting the same error, could pls try convert? maybe the file is not valid or something 

here is the file :
 
[7segments.zip](https://github.com/olikraus/u8g2/files/7838765/7segments.zip)

**Reply by olikraus** [reply]
i do not have access to the code, but I will try

**Reply by DanielRodrigues1** [reply]
if you can convert, could pls share the output file?

**Reply by DanielRodrigues1** [reply]
I mannaged to convert it oliver, i just  wanted to put a " . " character in the `7Segments_26x42` font, it works but only the dot i added make the characters overwrite themselves where the dot is, do you know how to fix it? maybe is something i have to do on fony? because all the other 
character that alreay was in the font works fine.

**Comment by olikraus** [comment]
> overwrite themselves where the dot is, do you know how to fix it?

Could this be an issue with the width of the character? If I remember correctly within fony you need to specify the delta where the next char begins.

**Reply by DanielRodrigues1** [reply]
i think its lacking a background color, its related to this delta? or there's other way to set a background on a char?

**Reply by DanielRodrigues1** [reply]
the char that i added is transparent,  like those fonts finishing with "t".

**Comment by olikraus** [comment, marked as answer]
hmm... I still do not understand the problem. 
But another guess: Maybe you need to use -b 1 mode with bdfconv.

**Reply by DanielRodrigues1** [reply]
It works!!!, -b 1 makes all background heigth equal?

**Reply by DanielRodrigues1** [reply]
Regardless, thanks for the attention oliver, you really helped me man.

**Reply by olikraus** [reply]
This is the intended purpose:
build modes:
```
 -b 0: Most compact, glyph bitmap is minimal
 -b 1: Like -b 0, but glyph bitmap is extended to the height of the largest glyph within the selected glyph list.
       Also the width of the gylphs is extended to cover the delta x advance.
 -b 2: Like -b 1, but glyph width is set to the width of the largest glyph within the selected gylph list.
 -b 3: Like -b 2, but width and height are forced to be a multiple of 8.
```
